examples/README.css

```diff
@@ -0,0 +1,13 @@
+/*
+ * this is loaded together with README.ipynb
+ * because it's named the same with css extension
+ */
+
+ /* ***** just a stupid setting that you cannot miss ***** */
+ div.cell.code_cell.rendered {
```

Comment:
you have an empty space at the beginning, I think...

Reply:
good catch